src/app.js

```diff
@@ -1 +1,20 @@
-// write code here
+const fs = require('fs');
+const path = require('path');
+
+let destination = process.argv[3];
```

Comment:
It's a good practice to validate the command-line arguments to ensure that both `sourceFile` and `destination` are provided. Consider adding a check to handle cases where these arguments might be missing.